client/src/index.css

```diff
@@ -18,5 +18,5 @@ body, html {
 }
 
 body {
-  background-color: #FDF6EC;
+  background-color: #eeeeee;

```

Comment:
Its too bad, because I don't like text on grey (feels like the very old browsers when page color was grey by default, and a grey page with text on it was a sign that it wasn't styled at all).

You could probably warm it up slightly?  🤷  No objections to the change, just noting its a tough one.

Reply:
The most recent shade was very nearly the least warm it could get, unfortunately with the blue banner the warm/cold clash just makes it way too immediately obvious that they don't match. Up to you I guess? I don't mind either one but it's tough with the banner

Reply:
The other thing is the warmer off-white is much more aggressive on the main page than you would think, so the tones are a bit off there if we want to do that